tools/mksurfdata_map/src/mksurfdat.F90

```diff
@@ -1186,9 +1156,17 @@ program mksurfdat
                 call abort()
              end if
           end do
+          
 
-          call change_landuse(ldomain, dynpft=.true.)
```

Comment:
- [ ] Comparing the changes in this PR with those done by @keerzhang1 in #1579, one difference I see is the removal of this change_landuse call in the time loop in this PR (it was left in place in #1579). It probably doesn't matter hugely, but I'm thinking that it's probably best to leave this call to change_landuse in place, as is done in #1579 .